reference/fleet/remote-elasticsearch-output.md

```diff
@@ -94,3 +94,82 @@ Remote clusters require access to the [{{package-registry}}](/reference/fleet/in
     3. Run the API request.
     4. Copy the encoded value of the generated API key.
     5. Back in the main cluster, paste the value you copied into the **Remote Kibana API Key** field.
+
+4. Choose whether uninstalled integrations should also be uninstalled on the remote cluster.
+
+### Troubleshooting
+
+When integration syncing is enabled for a remote {{es}} output, the current sync status is reported in **{{fleet}} Settings** in the **Outputs** table. To see a detailed breakdown of the integration syncing status, click on the **Integration syncing** status badge.
+
+You can also use the API to view the list of synced integrations with their sync status:
+
+1. In the main cluster, go to **{{fleet}} Settings** and edit the remote {{es}} output to check.
+
+2. Copy the output ID from the address in your browser.
+
+3. Go to **Management > Dev Tools**.
+
+4. Run the following query using the copied output ID:
+    ```sh
+    GET kbn:/api/fleet/remote_synced_integrations/<remote_output_id>/remote_status
+    ```
+    This should return the list of synced integrations with their sync status.
+
+::::{note}
+Syncing can take up to 5 minutes after an integration is installed, updated or removed on the main cluster.
```

Comment:
```suggestion
Syncing can take up to 5 minutes after an integration is installed, updated, or removed on the main cluster.
```
(We typically use an Oxford comma in the docs, but I'd guess we're pretty inconsistent about that. :-)  )